aspnetcore/fundamentals/middleware/index.md

```diff
@@ -191,12 +191,13 @@ ASP.NET Core ships with the following middleware components, as well as a descri
 | [HTTP Method Override](/dotnet/api/microsoft.aspnetcore.builder.httpmethodoverrideextensions) | Allows an incoming POST request to override the method. | Before components that consume the updated method. |
 | [HTTPS Redirection](xref:security/enforcing-ssl#require-https) | Redirect all HTTP requests to HTTPS (ASP.NET Core 2.1 or later). | Before components that consume the URL. |
 | [HTTP Strict Transport Security (HSTS)](xref:security/enforcing-ssl#http-strict-transport-security-protocol-hsts) | Security enhancement middleware that adds a special response header (ASP.NET Core 2.1 or later). | Before responses are sent and after components that modify requests (for example, Forwarded Headers, URL Rewriting). |
+| [MVC](xref:mvc/overview) | Processes requests with MVC/Razor Pages. | Terminal if a reqeust matches a route. |
```

Comment:
- reqeust --> request
- Thoughts on adding "(ASP.NET Core 2.0 or later)" next to the Razor Pages reference?